src/modules/Header/components/Header.module.css

```diff
@@ -0,0 +1,15 @@
+.headerBarUpper {
```

Comment:
I'm not sure if we should push these *.module.css files to GitHub because they are built from their source files (.pcss).

Reply:
Good question.